wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/fr.md

```diff
@@ -11,7 +11,8 @@ Les candidats doivent atteindre un seuil minimum de [kudosu](/wiki/Modding/Kudos
 Les moddeurs souhaitant postuler chez les Beatmap Nominators doivent remplir les critères suivants :
 
 - Ils ont accumulé au moins 200 [kudosu](/wiki/Modding/Kudosu) pour les moddeurs osu!, ou 150 [kudosu](/wiki/Modding/Kudosu) pour les moddeurs osu!taiko/osu!catch/osu!mania.
-- Ils fournissent 2 à 3 de leurs mods qui, selon eux, les qualifient pour faire partie des Beatmap Nominators.
+- Ils fournissent 3 de leurs mods qui, selon eux, les qualifient pour faire partie des Beatmap Nominators.
+  - Les anciens BN/NAT sont autorisés à soumettre au minimum 2 mods pour leur candidature.
```

Comment:
```suggestion
  - Les anciens BN/NAT sont autorisés à soumettre un minimum de 2 mods pour leur candidature.
```

Reply:
it just sounds better I think, apart from that, the French part seems to me correct